build.gradle

```diff
@@ -25,6 +25,7 @@ buildscript {
     ext.constraintlayout_version = '2.0.0-alpha5'
 
     repositories {
+        mavenLocal()
```

Comment:
Take this out?

Reply:
Yah, this should come out. It's super handy to have in while we're dealing with custom builds of android components.

Now we're moving to use `FeatureFlags`, I'll take it out. Once I pick this work up again (hi, @changecourse ), I'll work on my local machine.